src/db/course.ts

```diff
@@ -163,6 +163,7 @@ export const getNextVideo = async (currentVideoId: number) => {
     },
   });
   cache.set('getNextVideo', [currentVideoId.toString()], latestContent);
+  // console.log("---\nCourse.ts :", latestContent);
```

Comment:
You have a lot of console.log comments. That's really bad. Could you please resolve those?